src/platform_impl/windows/event_loop.rs

```diff
@@ -1200,7 +1200,7 @@ unsafe fn public_window_callback_inner<T: 'static>(
 
             let value = (wparam >> 16) as i16;
             let value = value as i32;
-            let value = value as f32 / winuser::WHEEL_DELTA as f32;
+            let value = -value as f32 / winuser::WHEEL_DELTA as f32;
```

Comment:
Please add a comment, mostly to draw attention to the fact that we're inverting the direction here (that minus is very small, yet quite significant).